packages/core/src/Stores/notification-store.js

```diff
@@ -291,12 +291,17 @@ export default class NotificationStore extends BaseStore {
                     is_10k_withdrawal_limit_reached &&
                     (needs_verification.includes('document') || document?.status !== 'verified');
                 const needs_poi = is_10k_withdrawal_limit_reached && identity?.status !== 'verified';
+                const needs_poinc =
+                    needs_verification.includes('income') && ['rejected', 'none'].includes(income?.status);
```

Comment:
can you access the status values from the utils file.
`income_status_codes.rejected, income_status_codes.none`